JamGame/Assets/Scripts/TileUnion/TileUnionImpl.cs

```diff
@@ -74,9 +74,20 @@ public void SetCoreModel(CoreModel coreModel)
         [SerializeField]
         private Vector2Int position;
 
-        [SerializeField, Range(0, 3)]
+        [Range(0, 3)]
+        [SerializeField]
         private int rotation;
 
+        [MinValue(0)]
+        [SerializeField]
+        private int projectedTilesCount;
```

Comment:
Please introduce some script that will do the following when assets are reloaded:
- store `projectedTilesCount`
- check that all the tiles where `tileToProject != null` contain proper amount of projected tiles
- calls `CreateProjectedTiles()` in case above condition is not met